app/policies/claims/school_policy.rb

```diff
@@ -0,0 +1,11 @@
+class Claims::SchoolPolicy < ApplicationPolicy
+  class Scope < ApplicationPolicy::Scope
+    def resolve
+      if user.support_user?
+        scope.all
+      else
+        scope.joins(:users).where(users: { id: user })
```

Comment:
It's a shame `user.schools` can't be used here 😢 that would have made for a nicer read.

Reply:
😢